android/java/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/MainActivity.java

```diff
@@ -47,6 +52,7 @@
 
     // Used for saving instance state
     private static final String STATE_IS_ANNOTATIONS_ON = "isAnnotationsOn";
+
```

Comment:
Seems they are due to the code formatting I used on Android Studio. Will fix it.